dwbb/src/interface/DW_gray_sync.scala

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2024 Jiuyang Liu <liu@jiuyang.me>
+package org.chipsalliance.dwbb.interface.DW_gray_sync
+
+import chisel3._
+import chisel3.experimental.SerializableModuleParameter
+import upickle.default
+
+object Parameter {
+  implicit def rw: default.ReadWriter[Parameter] =
+    upickle.default.macroRW[Parameter]
+}
+
+case class Parameter(
+    width: Int = 8,
+    offset: Int = 0,
+    regCountD: Boolean = true,
+    fSyncType: String = "2_stage_syn_ps",
+    tstMode: String = "no_latch",
+    verifEn: Int = 2,
```

Comment:
Same here.

Reply:
It would be clearer to be an `Int` type. There is no stander string to represent these values in the original documents.